user/transmission/template.py

```diff
@@ -0,0 +1,74 @@
+pkgname = "transmission"
+pkgver = "4.0.6"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DENABLE_DAEMON=ON",
+    "-DENABLE_GTK=ON",
+    "-DENABLE_QT=ON",
+    "-DENABLE_CLI=ON",
```

Comment:
-DUSE_SYSTEM_DEFLATE=ON \
		-DUSE_SYSTEM_EVENT2=ON \
		-DUSE_SYSTEM_MINIUPNPC=ON \
		-DUSE_SYSTEM_PSL=ON \

etc

Reply:
oh! didnt know that, thx!